spec/amp-actions-and-events.md

```diff
@@ -71,84 +71,123 @@ For example, the following is possible in AMP.
 ```
 
 ## Element Specific Events
+### form
```

Comment:
Also, please add ending punctuation to each description (ie., just needs a period if the sentence doesn't already have one).